shared/reducers/tracker2.tsx

```diff
@@ -106,4 +108,14 @@ export default Container.makeReducer<Actions, Types.State>(initialState, {
       ...rest,
     })
   },
+  [Tracker2Gen.userBlocked]: (draftState, action) => {
```

Comment:
You can handle the engine gen action itself and not translate it with a new tracker action plus saga here instead